requirements.txt

```diff
@@ -1 +1,2 @@
 Pillow>=9.2.0
+Requests>=2.31.0
```

Comment:
Not opposed to adding the `Requests` (capital, "R"?) package as a dependency. It is pretty "universal" package, where there has been talk of adding it to the standard library of Python. However, I am curious why this was needed instead of using `urllib.request.urlopen`? Again, not wrong, just curious.

Reply:
The `urllib.request.urlopen` actually works perfectly fine. I did not know it worked perfectly fine because I was using a url that was not working at all. When I was troubleshooting, I first changed that function before thinking to test the url. Requests was the last package I had tried before testing the url so it stayed.

Reply:
In that case, not to be annoying, but let's use `urllib.request.urlopen` and not add the `Requests` package. The fewer dependency the better.

Reply:
Change made as requested